CONTRIBUTING.rst

```diff
@@ -148,15 +149,158 @@ Bare in mind that your pull request might not necessarily get approved and furth
 Reviews are performed following the GitHub review mechanism, such that we can avoid many notifications for each comment made along the code. In general, reviewers will not pay too much attention to your pylint/pep/unittest changes (usually present in the 2nd commit), since those are supposed to be safe and not touching any algorithmically/logical parts of our baseline code.
 Last but not least, even if a PR has been recently closed and/or merged, feedback would still be welcomed.
 
+Creating unit tests
+-----------------------
+
+Test files need to be located under WMCore/test/python/ and
+need to mirror the structure you use for the packages under
+WMCore/src/python where every directory and test file is
+augmented with a _t and the class should be augmented with 'Test'
+If you develop for an external package you mimic this structure
+in the external package directory structure.
+
+E.g.: if you have a source package::
+
+   src/python/WMCore/DataStructs/LumiList.py
+
+you will have a test in the package::
+
+        test/python/WMCore_t/DataStructs_t/LumiList_t.py
```

Comment:
Too many spaces? Doesn't affect the preview format though.

Reply:
I do! The extra spacing just looked "ugly" in the source code, so I commented on it since it's easy to "fix", but not really important. You can leave it as is if you want.